src/services/car/leasing.ts

```diff
@@ -35,3 +36,35 @@ export const fetchLeasingFormUrl = async (args: {
       return null
   }
 }
+
+export const sendLeasingInterest = async (
```

Comment:
Can wen envelope the arguments for me this makes it always easier to understand the interface...
Same as for `fetchLeasingFormUrl`

Reply:
I wanted to keep it consistent with `sendMessageLead` since it'll be used in a similar context. We have a mix of both in the project let's tackle unifying the style within a tech-20 https://autoricardo.atlassian.net/browse/CAR-4826